src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -19,91 +19,110 @@ import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-erro
+```
+
+```
+Which infrastructure hosts are showing signs of degradation over the past hour?
+```
     </TabsPageItem>
 
     <TabsPageItem id="3">
-    Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
+New Relic AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle planned spikes, such as a big e-commerce sale, it's essential to understand the state of your system so you can improve it _ahead of time_.  Try asking New Relic AI:
 
-* How can I improve the performance of my infrastructure?
-* Were unique user sessions up or down last week compared to the week before?
+```
+How can I improve the performance of my infrastructure?
+```
+
+```
+Were unique user sessions up or down last week compared to the week before?
+```
     </TabsPageItem>
     </TabsPages>
-</Tabs>
+    </Tabs>
 
-### Get help creating NRQL queries [#NRQL-console]
+## Create NRQL queries [#NRQL-console]
 
-[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool to gain insights into systems. Still, when you're in a hurry, you might not have time to create the correct NRQL query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. 
 
 <img
-        title="Screenshot showing the Ask AI button in the New Relic platform"
-        alt="ask AI button"
-        src={askgrok}
-    />
+ title="Screenshot showing the Ask AI button in the New Relic platform"
+ alt="ask AI button"
+ src={askgrok}
+ />
 
+You can ask questions like:
 
-To enable NRAI in the NRQL console: 
+```
+How is this service performing?
+```
+```
+How has this service been performing over the past 24 hours?
+```
 
-1. Go to the user menu at the bottom left of the New Relic UI.
-2.  Click the <DoNotTranslate>**Features**</DoNotTranslate> tab.
-3. Scroll down to the <DoNotTranslate>**New Relic AI**</DoNotTranslate> section and click <DoNotTranslate>**Enable**</DoNotTranslate>.
+## Get help analyzing error logs [#errors]
 
-### Get help analyzing error logs [#errors]
+Error logs can be challenging to parse. New Relic AI helps you learn more about errors, query for related information, and ask about next steps. 
```

Comment:
@homelessbirds it sounds like you're using the word "parse" here as a synonym of "analyze." I'm asking because we have a preview feature that uses NRAI to generate parsing rules. When you're setting up parsing rules, you not "analyzing" per se. You're chopping up your logs into queryable chunks.

So, maybe you shouldn't use "parse" here, unless you mean that it applies to the generation of Grok query language and regular expressions that parse (divide) up logs.

Reply:
Yes, that makes sense. I can be more accurate with this.